test/Microsoft.ML.OnnxTransformTest/Microsoft.ML.OnnxTransformTest.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.Maml\Microsoft.ML.Maml.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.PipelineInference\Microsoft.ML.PipelineInference.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.Predictor.Tests\Microsoft.ML.Predictor.Tests.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML.Onnx.TestModels" Version="0.0.2-test" />
```

Comment:
Shouldn't these versions line up with the version of ML.NET?

Reply:
Not unless we think every version of ML.NET will require a corresponding update to the models, whether they've changed or not.

---
In reply to: [220277399](https://github.com/dotnet/machinelearning/pull/942#discussion_r220277399) [](ancestors = 220277399)

Reply:
Yup, agree with Tom. It's a test-model (data) package, which won't evolve with ML.Net versions.

See parallel example below for TF testmodels.

[https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.ML.TensorFlow.TestModels](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.ML.TensorFlow.TestModels)

---
In reply to: [220280718](https://github.com/dotnet/machinelearning/pull/942#discussion_r220280718) [](ancestors = 220280718,220277399)